clients/src/main/java/org/apache/kafka/clients/admin/internals/AdminApiHandler.java

```diff
@@ -57,13 +58,13 @@
      * Note that keys which received a retriable error should be left out of the
      * result. They will be retried automatically.
      *
-     * @param brokerId the brokerId that the associated request was sent to
+     * @param broker the broker that the associated request was sent to
```

Comment:
Not entirely sure about the new naming in place now, but does that still count as a broker?

Reply:
Well `process.roles` is documented like this in `config/kraft/README.md`:

* If `process.roles` is set to `broker`, the server acts as a broker in KRaft mode.
* If `process.roles` is set to `controller`, the server acts as a controller in KRaft mode.
* If `process.roles` is set to `broker,controller`, the server acts as both a broker and a controller in KRaft mode.

which suggests to me that we're using the term "broker"  to mean "a thing which handles Produce and Fetch etc". (However, "controller" is a bit confusing there, since while we might have several servers in the "controller" role only one will be  _the_ controller at any one time. I'm not aware of a good term for "server that is participating in the raft cluster, but might not be the current controller right now").

Reply:
Ok, thanks for the clarifications. I feel like `broker` is still fine here. Otherwise, maybe `node`?